aten/src/ATen/TensorUtils.cpp

```diff
@@ -298,7 +298,7 @@ std::vector<int64_t> defaultStrides(IntArrayRef sizes) {
   return strides;
 }
 
-int64_t computeStorageSize(IntArrayRef sizes, IntArrayRef strides) {
+int64_t computeStorageCapacity(IntArrayRef sizes, IntArrayRef strides, size_t itemsize) {
```

Comment:
It looks like you partially followed my suggestion to put bytes in the names? In that case, bytes here would be good too. (`computeStorageNbytes`, perhaps)